### PR TITLE
Bug fixes and minor changes to ExecutionTest

### PR DESCRIPTION
**Pull request by hekota**
Fixes 2 test bugs and a few razzle build warnings
Removes HlslTestUtils.h dependency on llvm/Support/Atomic.h
Brings it in sync with the HLK version of the tests

### CODE REVIEW

File: tools/clang/unittests/HLSL/HlslTestUtils.h [hunk 1]
```diff
@@ -235,6 +241,36 @@ inline bool ifdenorm_flushf_eq_or_nans(float a, float b) {
   return ifdenorm_flushf(a) == ifdenorm_flushf(b);
 }
 
+#else
+
+inline bool isdenorm(float f) {
+  return (std::numeric_limits<float>::denorm_min() <= f && f < std::numeric_limits<float>::min()) ||
+         (-std::numeric_limits<float>::min() <= f && f < -std::numeric_limits<float>::denorm_min());
```
Comment (MrTrillian) [inline tools/clang/unittests/HLSL/HlslTestUtils.h]: I think you need to swap the <= and < on the second line.

Comment (hekota) [reply tools/clang/unittests/HLSL/HlslTestUtils.h]: Yep

File: tools/clang/unittests/HLSL/ShaderOpTest.cpp [hunk 2]
```diff
@@ -717,7 +717,9 @@ void ShaderOpTest::CreateShaders() {
       CHECK_HR(pResult->GetResult(&pCode));
       CComPtr<IDxcBlobEncoding> pBlob;
       CHECK_HR(pCompiler->Disassemble((IDxcBlob *)pCode, (IDxcBlobEncoding **)&pBlob));
-      dxc::WriteUtf8ToConsoleSizeT((char *)pBlob->GetBufferPointer(), pBlob->GetBufferSize());
+      CComPtr<IDxcBlobEncoding> pUtf16Blob;
+      pLibrary->GetBlobAsUtf16(pBlob, &pUtf16Blob);
+      hlsl_test::LogCommentFmt(L"%*s", (int)pUtf16Blob->GetBufferSize() / 2, (LPCWSTR)pUtf16Blob->GetBufferPointer());
```
Comment (MrTrillian) [inline tools/clang/unittests/HLSL/ShaderOpTest.cpp]: / sizeof(wchar_t). This assumes that the buffer has no padding, and will probably log the null terminator if there's one, but that's probably no big deal.

Comment (hekota) [reply tools/clang/unittests/HLSL/ShaderOpTest.cpp]: This is just for debugging purposes, under #if 0 in shipping code.

**Pull request merged**